app/views/arclight/repositories/_in_person_repository.html.erb

```diff
@@ -1,3 +1,4 @@
+<%# Overrides arclight file . %>
```

Comment:
Let's also add why we override the file. And the link to the view components change that you put in the PR comment.